packages/gestalt-datepicker/dist/index.d.ts

```diff
@@ -420,6 +500,11 @@ export interface DateFieldProps {
  * =========================================================
  */
 
+/**
+ * https://gestalt.pinterest.systems/web/dateranger
```

Comment:
typo

Reply:
I wanted to be the 6th power ranger, DateRanger

Reply:
DONE